lib/stac-api/index.ts

```diff
@@ -100,44 +100,10 @@ export class PgStacApiLambda extends Construct {
         "allow connections from stac-fastapi-pgstac"
       );
     }
-
-    const stacApi = new apigatewayv2.HttpApi(
-      this,
-      `${Stack.of(this).stackName}-stac-api`,
-      {
-        defaultDomainMapping: props.stacApiDomainName
-          ? {
-              domainName: props.stacApiDomainName,
-            }
-          : undefined,
-        defaultIntegration: new apigatewayv2_integrations.HttpLambdaIntegration(
-          "integration",
-          this.stacApiLambdaFunction,
-          props.stacApiDomainName
-            ? {
-                parameterMapping:
-                  new apigatewayv2.ParameterMapping().overwriteHeader(
-                    "host",
-                    apigatewayv2.MappingValue.custom(
-                      props.stacApiDomainName.name
-                    )
-                  ),
-              }
-            : undefined
-        ),
-      }
-    );
-
-    this.url = stacApi.url!;
-
-    new CfnOutput(this, "stac-api-output", {
-      exportName: `${Stack.of(this).stackName}-url`,
-      value: this.url,
-    });
```

Comment:
I see that you have axed the `CfnOutputs` from these constructs - that's fine with me but just want to make sure it is intentional. I think leaving that up to the user to include in their CDK app is for the best anyways!

Reply:
I moved them into the API Gateway construct:

https://github.com/developmentseed/eoapi-cdk/blob/0c9d5a0d2bbbe0ccb5ff8488b220983603d39b69/lib/lambda-api-gateway/index.ts#L68-L71

I'm now questioning if that was the right call...